cfgov/jinja2/v1/_includes/macros/time.html

```diff
@@ -19,26 +19,26 @@
 
 {% macro render(datetime,
                 show_value_for={'date':true, 'time':true, 'timezone':true},
-                text_format=False) %}
+                text_format=False) -%}
 <span class="datetime">
-    {% if show_value_for.date == true  %}
+    {%- if show_value_for.date == true  -%}
     <time class="datetime_date" datetime="{{ dt.format_datetime(datetime) }}">
-      {{ dt.format_date(datetime, text_format) }}
+        {{- dt.format_date(datetime, text_format) -}}
     </time>
-    {% endif %}
+    {%- endif -%}
 
-    {% if show_value_for.date == true and show_value_for.time == true %}
+    {%- if show_value_for.date == true and show_value_for.time == true %}
     <span class="datetime_divider">@</span>
     {% endif %}
 
-    {% if show_value_for.time == true %}
+    {%- if show_value_for.time == true %}
     <time class="datetime_time" datetime="{{ dt.format_datetime(datetime) }}">
         {% if show_value_for.timezone == true %}
         {{ dt.format_time(localtime(datetime), show_timezone=true) }}
         {% else %}
         {{ dt.format_time(datetime) }}
         {% endif %}
     </time>
-    {% endif %}
+    {%- endif %}
```

Comment:
Is there a reason to omit the closing `-` with some blocks but not others? If not, maybe remove them all for consistency's sake.

Reply:
I tried to be as conservative as possible with changes to this file because I wasn't sure if, say, the newlines after closing `<span>` tags might affect how things are rendered.